locale/ko/about/working-groups.md

```diff
@@ -136,14 +134,14 @@ Core Working Groups are created by the
 <!--
 ### [Website](https://github.com/nodejs/nodejs.org)
 
-The website working group's purpose is to build and maintain a public
-website for the `Node.js` project.
+The Website Working Group's purpose is to build and maintain a public
+website for the Node.js project.
 
-Its responsibilities are:
-* Develop and maintain a build and automation system for `nodejs.org`.
-* Ensure the site is regularly updated with changes made to `Node.js` like
-releases and features.
-* Foster and enable a community of translators.
+Responsibilities include:
+* Developing and maintaining a build and automation system for nodejs.org.
+* Ensuring the site is regularly updated with changes made to Node.js, like
```

Comment:
Also please change \`nodejs.org\` to nodejs.org, \`Node.js` to Node.js in translation.